smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/HTTPBindingProtocolGenerator.kt

```diff
@@ -124,12 +121,12 @@ fun formatHeaderOrQueryValue(
 /**
  * Abstract implementation useful for all HTTP protocols
  */
-abstract class HttpBindingProtocolGenerator : ProtocolGenerator {
+abstract class HTTPBindingProtocolGenerator(
+    override val customizations: HTTPProtocolCustomizable,
+) : ProtocolGenerator {
```

Comment:
Renamed, and accepts customizations as a param to allow for Smithy protocol-specific tweaks to codegen.